.github/workflows/wasm-size.yml

```diff
@@ -1,14 +1,15 @@
 name: "QE: WASM size"
 on:
-  pull_request:
-    paths-ignore:
-      - ".github/**"
-      - "!.github/workflows/wasm-size.yml"
-      - ".buildkite/**"
-      - "*.md"
-      - "LICENSE"
-      - "CODEOWNERS"
-      - "renovate.json"
+  workflow_dispatch:
```

Comment:
Temporary, restored and fixed in #4688 
Can not be tested on the same PR because WASM size job compares sizes of PR with a base branch and at the moment, `main` and this branch built very differently with the artifacts at the different paths.

Reply:
:+1: I had to do this same hack in a recent PR.